include/backup_restore.hpp

```diff
@@ -12,6 +12,17 @@ namespace vpd
  * @brief class to implement backup and restore VPD.
  *
  */
+
+enum BackupAndRestoreStatus
+{
+    NotStarted = 0,
+    Initiated = 1,
```

Comment:
construction is basically "Instantiation" so "Initiated -> Instantiated" similarly "InitiationFailed -> InstantiationFailed".
Rest looks good to me.

Reply:
Updated.